core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -221,6 +221,10 @@ private boolean isBlitz() {
     return match.getModule(BlitzMatchModule.class) != null;
   }
 
+  private boolean isWool() {
+    return match.getModule(WoolMatchModule.class) != null;
+  }
+
```

Comment:
Being wool or not, shouldn't be the reason used to compact, See the `isCompactWool` method, what it defines is that wools should be compacted if they don't fit the scoreboard.